### ISSUE TITLE
[Feature Request] Add an option to not block all the comments

### ISSUE THREAD

**Issue by Archie-2021** [grantwinney/hide-comments-everywhere#106]
It is basically an option to have a blacklist of websites and allowing the rest. Also Adding an option to ignore/pause the rule temporarily for current tab (not changing the whitelist and blacklist for it).

**Comment by grantwinney** [comment]
The first request would mean redesigning the addon, since I wrote it assuming people would want to block most comments, and just let a few through.

The second request can be achieved just by clicking the speech bubble in the popup. That'll toggle comments for the current website.

![image](https://user-images.githubusercontent.com/1354059/167048515-9525aa06-7c9e-4486-b883-c4ac5743f5bc.png)

If you want it to only be temporary, and then hide comments again when you refresh the tab, there's a setting for that. Just uncheck the _"remember your setting for the site"_ option.

![image](https://user-images.githubusercontent.com/1354059/167048783-b0398914-92da-4e78-a8f9-6249d3b38942.png)

**Comment by Archie-2021** [comment]
You are right, but there is a use case for wanting occasional turning off comment section too, like wanting to "save the page" but not the comments or just a sudden need to navigate the page faster and focusing on the main text. 
I think a bit of it is just because I am new to it and not used to it, like yesterday I was wondering for couple of minutes why there is no comments on reddit pages I am viewing until remembering that I installed this plugin. 

Thanks you anyway

**Comment by grantwinney** [comment]
It'd be kinda cool to add a setting you could enable, that indicates the page has hidden comments... to avoid the issue like you had on Reddit. Maybe when I have more time I'll tackle that.

In the meantime, I guess give it a little while to get used to it, and then if you think of anything that would make it more intuitive let me know.
